SeaFightLogic/Ships/Ship.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaFight
+{
+
+    public abstract class Ship
+    {
+        public int Speed { get; protected set; }
+        public int ActionDistance { get; protected set; }
+        public int Size { get; protected set; }
+
+        public Ship(int size, int speed, int actionDistance)
+        {
+            if (size < 1 || size > 5)

```

Comment:
Why does size should be more than 5? It can be limited by game rules, but size limitation is not the responsibility of the ship.

Reply:
Now game rules are read from config file. Then there will be some static class "Game", from which these rules will be read